src/main/java/com/tma/teamhr/service/impl/TeamServiceImpl.java

```diff
@@ -40,6 +40,17 @@ public TeamResponseDTO getById(int id) {
         return new TeamResponseDTO(optionalTeam.get());
     }
 
+    @Override
+    public TeamResponseDTO update(TeamRequestDTO requestDTO) {
+        Optional<Team> optionalTeam = teamRepository.findById(requestDTO.getId());
+        if(optionalTeam.isEmpty())
+            throw new NullPointerException(message.NOTEXIST_ID);
```

Comment:
Why does it return NullPointerException?

Reply:
it my mistake, it should return ApiRequestException

Reply:
Fixed